### PR TITLE
Added Community Sensor Model as a dependency

### PR DESCRIPTION
**Pull request by scsides**
Modified cmake system to find CSM includes and library

## Description

Added the Community Sensor Model library and includes to the dependencies in preparation for adding CSM models as an option to ISIS

## How Has This Been Tested?

The output of the cmake configuration was checked to see that the lib and inc directories were found.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [X] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [X] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ dependencies:
   - bzip2
   - cmake >=3.15
   - cspice
+  - csm==3.0.3
```
Comment (jessemapel) [inline environment.yml]: For some reason this isn't resolving correctly https://astroservices.usgs.gov/jenkins/job/USGS-Astrogeology/job/ISIS3/job/PR-4130/1/execution/node/44/log/

**Comment by krlberry** [comment]
I know this isn't working on Jenkins but it builds fine for me locally. Is it okay if I merge this? It's not into dev, so it's not going to break anything.

File: environment.yml (hunk 1 above)
Comment (scsides) [reply environment.yml]: This is a feature branch, so we can merge as needed. I don't see a problem with merging for now. As soon as I figure out what is causing it, I'll put up another PR. I think I have a good idea.

Comment (scsides) [reply environment.yml]: I changed it to allow any 3.0.3.X version, and it now resolves the packages.

**Comment by jessemapel** [comment]
Builds are linking properly, I think we can merge this

**Review by jessemapel — APPROVED** [review]

**Pull request merged**